pkg/printer/workload_deliverable_printer.go

```diff
@@ -106,15 +106,18 @@ func DeliverableIssuesPrinter(w io.Writer, deliverable *cartov1alpha1.Deliverabl
 		return nil
 	}
 	printIssues := func(deliverable *cartov1alpha1.Deliverable, _ table.PrintOptions) ([]metav1beta1.TableRow, error) {
-		readyRow := metav1beta1.TableRow{
-			Cells: []interface{}{
-				fmt.Sprintf("Deliverable [%s]:", readyCondition.Reason),
-				readyCondition.Message,
-			},
+		rows := []metav1beta1.TableRow{}
+		if strings.TrimSpace(readyCondition.Message) != "" {
+			readyRow := metav1beta1.TableRow{
+				Cells: []interface{}{
+					fmt.Sprintf("Deliverable [%s]:", readyCondition.Reason),
```

Comment:
create const for kind `Deliverable` in pkg cartov1alpha1

Reply:
Addressed in PR #261